### PR TITLE
Created Blog for How do we build and push our custom docker images to a central repository?

### PR DESCRIPTION
**Pull request by Vrukshali-26**
Here I have mentioned two ways of creating Docker images.
1. Using commit command
2. Using Dockerfile

And then I have pushed this docker image to the DockerHub registry manually.
Since the blog went very much longer so I didn't mention the automated process with GitHub Actions.

### CODE REVIEW

**Review by cnu1812 — COMMENTED** [review]
.

File: resources/How_do_we_build_and_push_our_custom_docker_images_to_a_central_repository.md [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+  apr.x86_64 0:1.4.8-7.el7 apr-util.x86_64 0:1.5.2-6.el7 centos-logos.noarch 0:70.0.6-3.el7.centos httpd-tools.x86_64 0:2.4.6-97.el7.centos.4 mailcap.noarch 0:2.1.41-2.el7
+
+Complete!
+```
+
+### Step 5: Now create a webpage at location `/var/www/html/index.html`
+
+👉🏻 ```Custom docker image created !!!```
+
+### Step 6: Now exit from the running container using `exit` command. Exit command will stop the container. Exit command of docker same as power off or shut down of our computer.
```
Comment (cnu1812) [inline resources/How_do_we_build_and_push_our_custom_docker_images_to_a_central_repository.md:158]:
```suggestion
### Step 6: Now exit from the running container using `exit` command.

 * Exit command will stop the container. Exit command of docker same as power off or shut down of our computer.
```
This line explains about exit command, so if it is in a new line it's quiet comfortable for readers.

**Review by cnu1812 — COMMENTED** [review]
Please have a look into it!

**Comment by AbhijithGanesh** [comment]
@Vrukshali-26 please keep the commit levels low from next PR

File: resources/How_do_we_build_and_push_our_custom_docker_images_to_a_central_repository.md [hunk 2]
```diff
@@ -0,0 +1,426 @@
+# 🔰 How do we build and push our custom docker images to a central repository? 
+
+## Docker 🐋
+Docker is an open-source lightweight virtualization tool. It is containerizing platform in which user can run and deploy application and its dependencies and form containers to run over any linux infrastructure.
+
+## What Exactly Is A Docker Image? 🐋
+A Docker image is a read-only, inert template that comes with instructions for deploying containers. In Docker, everything basically revolves around images.An image consists of a collection of files (or layers) that pack together all the necessities—such as dependencies, source code, and libraries—needed to set up a completely functional container environment.
+
+<b>Images are stored on a Docker registry, such as the Docker Hub, or on a local registry.</b>
```
Comment (AbhijithGanesh) [inline resources/How_do_we_build_and_push_our_custom_docker_images_to_a_central_repository.md:9]: Use `**<Enclosing text>** to`  **Bolden** text

File: resources/How_do_we_build_and_push_our_custom_docker_images_to_a_central_repository.md [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+## 🔰 Let's take an example and built the docker image using both the ways:
+
+👉🏻 So here we are going to <b>Build Apache Web Server Docker Image</b>
+
+<i>Apache web server is popular open source http web server tool which is widely used for deployment of webpages. It can be installed in any operating system.</i>
+
+## 1. Manual :
+
+### Step 1: Get the latest Centos Docker image by using <b>docker pull command.</b><br> 
+`docker pull centos:7` <i>Docker pull command is used to download or pull latest image from Docker Hub repositories.</i>
```
Comment (AbhijithGanesh) [inline resources/How_do_we_build_and_push_our_custom_docker_images_to_a_central_repository.md:27]: I think you should choose `centos:latest`

File: resources/How_do_we_build_and_push_our_custom_docker_images_to_a_central_repository.md [hunk 4]
```diff
@@ -0,0 +1,426 @@
...
+### Step 2: To check and list all docker images `docker images`
+
+🔅 Output :
+```
+REPOSITORY   TAG       IMAGE ID       CREATED        SIZE
+centos       7         eeb6ee3f44bd   4 months ago   204MB
+```
+
+### Step 3: To run docker image we use following command <br>
+👉🏻 `-it` : This argument is used to allocate a bash shell and take standard input. <br>
```
Comment (AbhijithGanesh) [inline resources/How_do_we_build_and_push_our_custom_docker_images_to_a_central_repository.md:47]: It'd be appropriate if you can explain the tags like this

`-i` gives an option to set the cli flag to interactive and `-t` is used to set the terminal flag.

**Comment by AbhijithGanesh** [comment]
This looks good to me otherwise

**Review by AbhijithGanesh — APPROVED** [review]

File: resources/How_do_we_build_and_push_our_custom_docker_images_to_a_central_repository.md (hunk 3 above)
Comment (Vrukshali-26) [reply resources/How_do_we_build_and_push_our_custom_docker_images_to_a_central_repository.md:27]: At first, I took centos:latest but the image had some connectivity problems like downloading packages, so I changed it to 7 and it worked fine for me! Using centos:latest will work as well.

**Comment by Vrukshali-26** [comment]
I have made the mentioned changes @AbhijithGanesh @cnu1812

File: resources/How_do_we_build_and_push_our_custom_docker_images_to_a_central_repository.md (hunk 3 above)
Comment (AbhijithGanesh) [reply resources/How_do_we_build_and_push_our_custom_docker_images_to_a_central_repository.md:27]: Its fine then

**Review by AbhijithGanesh — APPROVED** [review]

**Pull request merged**